### DISCUSSION TITLE
Understanding the Plugin working

### DISCUSSION THREAD

**Discussion by casual-coderr** [remotely-save/remotely-save#348] [Q&A]
Hi,
I'm just a newbie and I wanted to understand how this plugin works. 
Consider the following example, I have two devices and 100 files in my vault. So after the initial sync in both, If I change a file and click on sync on my desktop, in my S3 would I have 101 reads and 2 write ? (1 extra read and write for the config file in cloud)
or would I have only 2 read and 2 write. (1 read and write for config file and one more read and write for the file).

And again while syncing in mobile would I have the same 101 reads and 1 write (1 write for config update)?

For a bigger vault, I don't want to go over my monthly free s3 limits.

**Comment by fyears** [comment]
it's complicated and not so ideal for "101 reads for 100 files".

the plugin will: read the list of all files, compare timestamps, download and upload some files if some timestamps are different, etc. all the operations will cost the api calls, and will be definitely larger than "101 reads for 100 files".
